operations/utility_storage/Add_Ceph_Node.md

```diff
@@ -136,10 +136,10 @@
      10.252.1.13
      ```
 
-     Update the existing HAproxy config on `ncn-s001` to include the added node.
+     Update the existing HAproxy config on `ncn-s001`, `ncn-s002` or `ncn-s003` to include the added node.
 
      ```
-     ncn-s001# vi /etc/haproxy/haproxy.cfg
+     ncn-s# vi /etc/haproxy/haproxy.cfg
```

Comment:
This is specific to the node you are adding back.   ncn-s# indicated you can run that from any node..  not sure how to address this.  also one on line 163.